src/ontology/uberon-idranges.owl

```diff
@@ -17,260 +16,269 @@ AnnotationProperty: allocatedto:
 
 AnnotationProperty: idprefix:
 
-AnnotationProperty: iddigits: 
+AnnotationProperty: iddigits:
```

Comment:
Reviewers: all the changes in the `uberon-idranges.owl` files are due to the fact that the file has been re-serialized as a side-effect of the addition of the two new ranges. Those serialisation-induced changes are inconsequential.